kubernetes-client/src/test/resources/test-ec-kubeconfig

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: testns/ca.pem
+    server: https://172.28.128.4:8443
+  name: 172-28-128-4:8443
+contexts:
+- context:
+    cluster: 172-28-128-4:8443
+    namespace: default
+    user: user/172-28-128-4:8443
+  name: default/172-28-128-4:8443/user
+current-context: default/172-28-128-4:8443/user
+kind: Config
+preferences: {}
+users:
+- name: user/172-28-128-4:8443
+  user:
+    client-certificate-data: -----BEGIN CERTIFICATE-----
```

Comment:
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            fe:58:d1:51:27:e3:72:de:a4:82:21:18:18:19:0a:6e
    Signature Algorithm: sha256WithRSAEncryption
        Issuer:
            commonName                = minikubeCA
        Validity
            Not Before: Nov  1 09:22:18 2021 GMT
            Not After : Nov  1 09:22:18 2022 GMT
        Subject:
            commonName                = uzair
            organizationName          = developers
        Subject Public Key Info:
            Public Key Algorithm: id-ecPublicKey
                Public-Key: (256 bit)
                pub: 
                    04:90:03:03:dd:69:41:36:c3:74:b6:23:42:65:20:
                    bc:80:af:f9:3b:8a:f1:9a:48:23:f7:9d:e4:bd:68:
                    2f:d7:5f:c0:a2:b8:31:0c:56:69:af:6f:9a:cd:82:
                    40:37:a6:4d:b7:c8:57:a0:19:6c:ac:1b:43:0f:32:
                    a7:7c:f2:6c:ce
                ASN1 OID: prime256v1
        X509v3 extensions:
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage: 
                TLS Web Client Authentication
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Authority Key Identifier: 
                keyid:E2:AC:9C:59:3E:6A:ED:CF:F8:3A:05:88:1D:D2:88:2F:10:61:8D:2F

    Signature Algorithm: sha256WithRSAEncryption
         d4:e8:99:2e:00:2f:8c:f4:73:17:74:29:94:21:b6:cd:80:0e:
         56:e0:25:05:26:23:b4:d4:f3:e2:3b:d2:aa:10:85:db:c5:0b:
         7c:45:b9:03:37:28:75:50:92:de:1a:9a:3f:9a:87:3e:f8:64:
         c1:81:f0:51:fb:48:7b:dd:2d:d6:fd:c8:8b:19:40:9b:6d:c3:
         a3:15:ca:b0:b7:fc:66:af:44:14:0c:dc:ac:00:07:40:95:30:
         44:1f:44:bf:cf:61:35:c9:4a:63:e2:89:c4:07:f4:12:d4:5b:
         d4:73:27:03:f8:82:72:25:12:3d:b2:8c:b9:c0:3f:28:a5:1f:
         82:d2:88:1b:e3:13:dd:e7:3c:11:dd:63:44:85:fe:fd:7f:27:
         2d:e9:9e:97:22:87:4e:5e:1d:a8:45:b4:ca:c4:dc:51:f5:61:
         a5:42:2f:69:dc:50:68:9f:ff:fc:7d:4c:d8:51:ed:65:f6:c3:
         4f:97:f9:fa:98:e3:0d:10:e3:19:89:08:65:bf:0e:f6:2a:04:
         6a:9a:bd:d1:18:53:51:4c:5e:f9:61:a9:09:11:b9:e6:0e:19:
         b5:3c:3f:65:65:85:4e:6a:77:45:9d:7d:d8:3c:16:89:24:76:
         8c:cb:45:48:24:09:b7:30:c4:e4:de:6b:52:e2:b7:68:ae:0c:
         6c:c9:7b:32
-----BEGIN CERTIFICATE-----
MIICUDCCATigAwIBAgIRAP5Y0VEn43LepIIhGBgZCm4wDQYJKoZIhvcNAQELBQAw
FTETMBEGA1UEAxMKbWluaWt1YmVDQTAeFw0yMTExMDEwOTIyMThaFw0yMjExMDEw
OTIyMThaMCUxEzARBgNVBAoTCmRldmVsb3BlcnMxDjAMBgNVBAMTBXV6YWlyMFkw
EwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEkAMD3WlBNsN0tiNCZSC8gK/5O4rxmkgj
953kvWgv11/AorgxDFZpr2+azYJAN6ZNt8hXoBlsrBtDDzKnfPJszqNWMFQwDgYD
VR0PAQH/BAQDAgWgMBMGA1UdJQQMMAoGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAw
HwYDVR0jBBgwFoAU4qycWT5q7c/4OgWIHdKILxBhjS8wDQYJKoZIhvcNAQELBQAD
ggEBANTomS4AL4z0cxd0KZQhts2ADlbgJQUmI7TU8+I70qoQhdvFC3xFuQM3KHVQ
kt4amj+ahz74ZMGB8FH7SHvdLdb9yIsZQJttw6MVyrC3/GavRBQM3KwAB0CVMEQf
RL/PYTXJSmPiicQH9BLUW9RzJwP4gnIlEj2yjLnAPyilH4LSiBvjE93nPBHdY0SF
/v1/Jy3pnpcih05eHahFtMrE3FH1YaVCL2ncUGif//x9TNhR7WX2w0+X+fqY4w0Q
4xmJCGW/DvYqBGqavdEYU1FMXvlhqQkRueYOGbU8P2VlhU5qd0Wdfdg8FokkdozL
RUgkCbcwxOTea1Lit2iuDGzJezI=
-----END CERTIFICATE-----